test/helper.js

```diff
@@ -7,6 +7,12 @@ const pid = process.pid
 const hostname = os.hostname()
 const v = 1
 
+const isWin = process.platform === 'win32'
+
+function getPathToNull () {
+  return isWin ? '\\\\.\\NUL' : '/dev/null'
+}
```

Comment:
Is that file actually /dev/null? Or we should remove it before exit?

Reply:
It is, indeed, null: https://stackoverflow.com/questions/11053977/how-can-i-write-to-the-nul-device-under-windows-from-node-js